app/src/main/java/com/appunite/loudius/ui/components/LoudiusPullToRefreshBox.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 AppUnite S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:OptIn(ExperimentalMaterialApi::class)
+
+package com.appunite.loudius.ui.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.pullrefresh.PullRefreshIndicator
+import androidx.compose.material.pullrefresh.PullRefreshState
+import androidx.compose.material.pullrefresh.pullRefresh
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun LoudiusPullToRefreshBox(
+    pullRefreshState: PullRefreshState,
+    refreshing: Boolean,
```

Comment:
nitpick: I don't really like passing `refreshing` state twice to this state. I mean about:
```kotlin
LoudiusPullToRefreshBox( 
    content = content,
    refreshing = false, // <--
    pullRefreshState = rememberPullRefreshState(
                refreshing = false, // <--
                onRefresh = {},
            ),
)
```

It's fine, but I'd love having this a little bit simpler ;) Passing the same state twice, is always a place for possible issues.

I wonder if this is possible to just pass:
```kotlin
LoudiusPullToRefreshBox( 
    content = content,
    refreshing = false,
    onRefresh = {},
)
```

or:

```kotlin
LoudiusPullToRefreshBox( 
    content = content,
    pullRefreshState = rememberPullRefreshState(
                refreshing = false,
                onRefresh = {},
            ),
)
```

Reply:
I like it, done🤓